tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +115,10 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $containerBuilderProphecy = $this->prophesize(ContainerBuilder::class);
+
+        $kernelProphecy->getContainer()->willReturn($containerBuilderProphecy);
```

Comment:
BTW do you want to add a test with has-> false in order to trigger the exception ?